translations/strings.yaml

```diff
@@ -709,6 +724,9 @@ subscriptionManagement:
   addFirefoxRelay:
     value: Add Firefox Relay
     comment: Block of text appearing above a link to learn more about the VPN / Relay bundle offering.
+  valueProp1: No logging of your network activity
```

Comment:
Please add comments for these 3 strings to explain how/where they're used.

Also, I assume these are not 2.14 at this point?

Reply:
These are 2.15 strings, correct